### DISCUSSION TITLE
skip the "select the product to launch" on epic games

### DISCUSSION THREAD

**Discussion by y2kmedman** [rfvgyhn/min-ed-launcher#107] [Q&A]
I want to be able to skip or set a default on the part where the min-ed-launcher asks "select the product to launch" to where it always starts Odyssey automatically. I installed this to minimize the steps to start the game and I don't have to enter Odyssey as a choice on my steam version and I love that. How can I get the Epic games version to do the same? Does it help to know I am using a newer version of min-ed-launcher on the epic games setup?

**Comment by rfvgyhn** [comment]
Yeah, should be the same flags as your steam version. `/autostart /autoquit /edo`. You can follow the [Epic steps](https://github.com/rfvgyhn/min-ed-launcher#epic) if you're unsure where to specify them.

**Reply by y2kmedman** [reply]
I did read the reamme.md file and here on Github and followed the instructions. I am really grateful for this free software.
